modules/mono/mono_gd/gd_mono.h

```diff
@@ -109,7 +109,7 @@ class GDMono {
 	LoadedApiAssembly core_api_assembly;
 	LoadedApiAssembly editor_api_assembly;
 
-	typedef bool (*CoreApiAssemblyLoadedCallback)();
+	using (*CoreApiAssemblyLoadedCallback)() = bool;
```

Comment:
This seems wrong. Looking at the other changes I think this should be like this?:

```suggestion
	using CoreApiAssemblyLoadedCallback = bool (*)();
```